database_persistence.rb

```diff
@@ -17,6 +19,28 @@ def query(statement, *params)
     @db.exec_params(statement, params)
   end
 
+  def user_by_id(id)
+    sql = <<~SQL
+      SELECT
+        u.slackname,
+        u.firstname,
+        u.lastname,
+        u.about_me,
+        t.name AS track,
+        c.code AS course,
+        ti.name AS timezone
+      FROM users u
+        JOIN tracks t ON u.track_id = t.id
+        JOIN courses c ON u.course_id = c.id
+        JOIN timezones ti ON u.timezone_id = ti.id
+      WHERE u.id = $1
+    SQL
+
+    result = query(sql, id)
+    tuple = result.first
+    tuple_to_user(tuple)
+  end
+
   def first_user()
```

Comment:
Same comment as in peers.rb file. This method is for testing purposes only, right?

Reply:
Yep!